File: 30 Days of Code/Day 05 Loops/code.py
# Written: 08-Jan-2020
# https://www.hackerrank.com/challenges/30-loops

#!/bin/python3

import math
import os
import random
import re
import sys

if __name__ == '__main__':
    n = int(input())
    for i in range(1, 11):
        print("{} x {} = {}".format(n, i, n*i))

File: 30 Days of Code/Day 05 Loops/code.kt
// Written: 07-Apr-2020
// https://www.hackerrank.com/challenges/30-loops/problem

import java.io.*
import java.math.*
import java.security.*
import java.text.*
import java.util.*
import java.util.concurrent.*
import java.util.function.*
import java.util.regex.*
import java.util.stream.*
import kotlin.collections.*
import kotlin.comparisons.*
import kotlin.io.*
import kotlin.jvm.*
import kotlin.jvm.functions.*
import kotlin.jvm.internal.*
import kotlin.ranges.*
import kotlin.sequences.*
import kotlin.text.*

fun solution(n: Int) {
    for (i in 1..10) println("$n x $i = ${n*i}")
}

fun main(args: Array<String>) {
    val scan = Scanner(System.`in`)
    val n = scan.nextLine().trim().toInt()

    solution(n)
}
